app/cdap/components/AddTransformation/styles.ts

```diff
@@ -0,0 +1,278 @@
+      fontWeight: 400,
+      fontSize: 15,
+      lineHeight: '26px',
+      letterSpacing: '0.46px',
+      color: '#FFFFFF',
+      alignSelf: 'flex-end',
+      marginTop: 30,
+      textTransform: 'none',
+      marginBottom: 20,
+      marginRight: 8,
+    },
+    buttonStyles: {
+      '&:hover': {
+        backgroundColor: '#3994FF',
+      },
+    },
+    selectColumnsHeaderStyles: {
+      display: 'flex',
+      justifyContent: 'space-between',
+    },
+    recipeStepsTableHeadStyles: {
+      padding: 10,
+      fontStyle: 'normal',
+      fontWeight: 600,
+      fontSize: 16,
+      lineHeight: '150%',
+      letterSpacing: '0.15px',
+      color: '#5F6368',
+    },
+    recipeStepsTableRowStyles: {
+      fontStyle: 'normal',
+      fontWeight: 400,
+      fontSize: 16,
+      lineHeight: '150%',
+      letterSpacing: '0.15px',
+      color: '#5F6368',
+    },
+    domm: {
+      width: '100%',
+    },
+    recipeStepsTableBodyRowStyles: {
+      '&:hover': {
+        background: '#EFF0F2',
+        '& td:last-child': {
+          visibility: 'visible',
+        },
+      },
+    },
+    recipeStepsActionTypeStyles: {
+      fontWeight: 400,
+      maxWidth: 190,
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      fontSize: 14,
+      fontStyle: 'normal',
+      marginBottom: 5,
+      textTransform: 'capitalize',
+    },
+    displayNone: {
+      visibility: 'hidden',
+    },
+    recipeStepsDeleteStyles: {
+      width: 18,
+      height: 20,
+      cursor: 'pointer',
+      //   padding: '15px 10px',
+    },
+    radioStyles: {
+      '& span:last-child': {
+        fontStyle: 'normal',
+        fontWeight: 400,
+        fontSize: 14,
+        color: '#5F6368',
+      },
+    },
+    replaceWithInput: {
+      width: '90%',
+    },
+    replaceWithText: {
+      fontStyle: 'normal',
+      fontWeight: 400,
+      fontSize: 12,
+      color: '#5F6368',
+    },
+    searchFormControl: {
+      position: 'relative',
+      display: 'flex',
+    },
+    searchTextField: {},
+    searchInputAdornment: {
+      zIndex: 0,
+      cursor: 'pointer',
+      position: 'absolute',
+      right: 0,
+    },
+    isFocused: {
+      border: 'none',
+      borderBottom: '1px solid grey',
+      outline: 'none',
+    },
+    isBlurred: {
+      border: 'none',
+      borderBottom: '1px solid transparent',
+    },
+    circularProgress: {
+      transform: 'rotate(180deg) !important',
+    },
+    circularProgressSuccess: {
+      color: '#8BCC74',
+    },
+    circularProgressRed: {
+      color: '#E97567',
+    },
+    circularBarCell: {},
+    nullValueHead: {},
+    displayStyles: {
+      display: 'flex',
+      justifyContent: 'space-between',
+    },
+    tabledisplayStyles: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    columnstyles: {},
+    barStyles: {
+      paddingRight: 110,
+      height: 70,
+    },
+    divStyles: {
+      width: 200,
+    },
+
+    infoIcon: {
+      color: blue[500],
```

Comment:
why do we have so many classes in this file
are they all related to index.tsx or did we add sub components styles also here only
in that case, plz move them to respective sub components' styles.ts